docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       - api
 
   api:
-    image: silintl/idp-pw-api:develop
+    image: silintl/idp-pw-api:featurewebauthn
```

Comment:
Just so we don't forget, I think we'll want to set this back to `develop` before actually merging this PR.